### PR TITLE
Feat/analytics

### LINKED ISSUE #79: Plugin request: Google Analytics
### Basic example

At Gridsome.org we use vue-analytics and add this to `main.js``

```
import VueAnalytics from 'vue-analytics'

export default function (Vue, { head, router, isServer }) {
  Vue.use(VueAnalytics, {
    id: 'UA-127625720-1',
    disabled: isServer,
    debug: {
      sendHitTask: process.env.NODE_ENV === 'production'
    },
    router
  })
}
```

This could be done in a plugn instead that lets users add options in `gridsome.config.js`.

A plugin can use the Client API for this: https://gridsome.org/docs/client-api

### PR DESCRIPTION
**Pull request by EldoranDev**
This PR Adds the @gridsome/plugin-analytics plugin to gridsome.

This should fix #79

### CODE REVIEW

**Comment by tomtev** [comment]
Looks good :) Could you add a link to the Vue Analytics docs in the readme?

**Comment by tomtev** [comment]
Could you also rename the folder to plugin-google-analytics?

**Comment by EldoranDev** [comment]
> Could you also rename the folder to plugin-google-analytics?
> Looks good :) Could you add a link to the Vue Analytics docs in the readme?

Done. and Done

**Comment by hjvedvik** [comment]
Thanks, @EldoranDev :)

Here are a few more things that should be done:

- Clear or delete `CHANGELOG.md` (will be autogenerated)
- Add `homepage` and `repository` to `package.json`
- Set version as `0.0.0` (will be published as `0.0.1` or `0.1.0`)
- Run `yarn run lint` from the root folder

You can also remove all test projects if you have any, then run `yarn` in the root folder to commit changes in the `yarn.lock` file if any.

File: packages/plugin-google-analytics/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# @gridsome/plugin-analytics
+
+> Adds Google Analytics to the Page
```
Comment (hjvedvik) [inline packages/plugin-google-analytics/README.md]:
```suggestion
> Google Analytics plugin for Gridsome
```

File: packages/plugin-google-analytics/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+- `yarn add @gridsome/plugin-analytics`
+- `npm install @gridsome/plugin-analytics`
+
+## Usage
+
+```js
+module.exports = {
+  plugins: [
+    {
+      use: '@gridsome/plugin-analytics',
```
Comment (hjvedvik) [inline packages/plugin-google-analytics/README.md]:
```suggestion
      use: '@gridsome/plugin-google-analytics',
```

File: packages/plugin-google-analytics/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# @gridsome/plugin-analytics
+
+> Adds Google Analytics to the Page
+
+See [VueAnalytics](https://github.com/MatteoGabriele/vue-analytics/blob/master/README.md) for possible options.
+
+## Install
+- `yarn add @gridsome/plugin-analytics`
```
Comment (hjvedvik) [inline packages/plugin-google-analytics/README.md]:
```suggestion
- `yarn add @gridsome/plugin-google-analytics`
```

File: packages/plugin-google-analytics/README.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+- `npm install @gridsome/plugin-analytics`
```
Comment (hjvedvik) [inline packages/plugin-google-analytics/README.md]:
```suggestion
- `npm install @gridsome/plugin-google-analytics`
```

**Pull request merged**